server/src/app/config.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2021, Pyronear contributors.
+
+# This program is licensed under the GNU Affero General Public License version 3.
+# See LICENSE or go to <https://www.gnu.org/licenses/agpl-3.0.txt> for full license details.
+
+
+import os
+
+PROJECT_NAME: str = 'Pyronear local system API'
+PROJECT_DESCRIPTION: str = 'API for pyronear systems interactions on sites'
```

Comment:
Perhaps Webserver would be more accurate

Reply:
Pyronear local webserver?